src/popup/components/AccountDetailsBase.vue

```diff
@@ -206,14 +189,20 @@ export default defineComponent({
 @use '../../styles/mixins';
 @use '../../styles/typography';
 
+:deep(.ion-padding) {
+  background-color: variables.$color-bg-4;
+}
```

Comment:
The team agreed that we should not use `:deep` selector. If you want to influence the inner elements of child components please use bool prop or variant prop on that component and change the styling within the component when the prop changes.